_chapters/chapter_02.md

```diff
@@ -11,15 +11,15 @@ layout: rails_tutorial
 
 {% steps %}
 {% list %}
-  1.  Open Terminal and `cd` to `Projects/bookstore` from your home directory.
-
-      If you have Terminal open from the previous chapter, you should already be in `HOME/Projects/bookstore`. Remember how to check what directory you're in?
-
+  1.  If you have Terminal open from the previous chapter, you should already be in `HOME/Projects/bookstore`. Remember how to check what directory you're in?
```

Comment:
I like this better than what we had, but I think it could be a *little* more concise. I might revise it later.